src/core/App.module.scss

```diff
@@ -1,22 +1,28 @@
 .App {
   text-align: center;
-}
 
-.App-logo {
-  height: 40vmin;
-}
+  &-header {
+    background-color: #282c34;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+    color: white;
+  }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
+  &-logo {
+    height: 40vmin;
+  }
 
-.App-link {
-  color: #09d3ac;
-}
+  &-link {
+    color: #09d3ac;
+  }
+
+  &-grid {
+    max-width: 1300px;
+    margin: 0 auto;
+    display: flex;
+  }
+}
```

Comment:
Please add line breaks to the end of your files.

It seems unnecessary, but editors and GitHub prefer it. To get technical, the definition of a line of text requires a line break.

Reply:
Thanks @mshelley-twg for the context! I never knew the reason why end of file line breaks were necessary.